osutil/disks/disks_linux.go

```diff
@@ -543,11 +562,15 @@ func diskFromMountPointImpl(mountpoint string, opts *Options) (*disk, error) {
 			return nil, fmt.Errorf("cannot find disk for partition %s: incomplete udev output missing required property \"ID_PART_TABLE_UUID\"", partMountPointSource)
 		}
 
-		schema := realDiskProps["ID_PART_TABLE_TYPE"]
+		schema := strings.ToLower(realDiskProps["ID_PART_TABLE_TYPE"])
 		if schema == "" {
 			return nil, fmt.Errorf("cannot find disk for partition %s: incomplete udev output missing required property \"ID_PART_TABLE_TYPE\"", partMountPointSource)
 		}
 
+		if schema != "gpt" && schema != "dos" {
```

Comment:
oh hmm the changes right here might make this PR conflict with https://github.com/snapcore/snapd/pull/10919, oh well I suppose I can sort out the conflicts depending on which one lands first

Reply:
this conflict has been resolved